.github/workflows/package.yml

```diff
@@ -0,0 +1,99 @@
+name: Package Application
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    steps:
+      - name: Retrieves project from the git branch
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat package.json | grep '\"node\":' | sed 's/^ *//;s/ *$//;s/\"node\":\ \"//;s/\",//' )"
+        id: nvm
```

Comment:
This command doesn't seem to read anything from .nvmrc, at least from my understanding of it. We can just hardcode a Node version like we did in the CI workflow: https://github.com/MyCryptoHQ/signer/blob/master/.github/workflows/ci.yml#L17-L18

Reply:
You're right. Took it from the CI workflow before its rework - Removing :)